clang/docs/ReleaseNotes.rst

```diff
@@ -394,6 +394,7 @@ Bug Fixes to C++ Support
   expression references to an entity declared outside of the lambda. (#GH64808)
 - Clang's __builtin_bit_cast will now produce a constant value for records with empty bases. See:
   (#GH82383)
+- Fix a crash in lambda instantiation that missing set ``ThisType`` when checking capture. Fixes (#GH85343).
```

Comment:
```suggestion
- Fix a crash when instantiating a lambda that captures ``this`` outside of its context. Fixes (#GH85343).
```

Reply:
This sentence is really hard to parse; maybe something like this makes it a bit clearer what the actual change here is; if you can think of a better way of wording it, feel free to go w/ that instead.